components/pool-usage-pattern.tsx

```
@@ -0,0 +1,241 @@
+  weekdayPeakHoursPerDay: 8,
+  weekdayOffPeakConcurrentUsers: 10,
+  weekdayPeakConcurrentUsers: 80,
+  
+  // Weekend defaults (2 days, 4 peak hours per day, 40% utilization during peak)
+  weekendDaysCount: 2,
+  weekendPeakHoursPerDay: 4,
+  weekendOffPeakConcurrentUsers: 5,
+  weekendPeakConcurrentUsers: 40
+}
+
+export function PoolUsagePattern({ value = DEFAULT_USAGE_PATTERN, onChange, className }: PoolUsagePatternProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PoolUsagePattern'. Consider to delete it or rename it.

'PoolUsagePattern' is defined here:


(lint/suspicious/noRedeclare)